src/components/Header/UserPreferences/Automation/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { ReactElement, useEffect, useState } from 'react'
+import { useAutomation } from '../../../../@context/Automation/AutomationProvider'
+import Tooltip from '../../../@shared/atoms/Tooltip'
+import Details from './Details'
+import Transaction from '@images/transaction.svg'
+import Caret from '@images/caret.svg'
+import Lock from '@images/lock.svg'
+import classNames from 'classnames/bind'
+import stylesIndex from '../index.module.css'
+import styles from './index.module.css'
+
+const cx = classNames.bind(styles)
+
+export default function Automation(): ReactElement {
+  const {
+    autoWallet,
+    isAutomationEnabled,
+    hasValidEncryptedWallet,
+    balance,
+    nativeBalance
+  } = useAutomation()
+
+  const [hasZeroNetworkTokenBalance, setHasZeroNetworkTokenBalance] =
+    useState<boolean>(false)
+  const [hasZeroERC20TokenBalances, setHasZeroERC20TokenBalances] =
+    useState<boolean>(false)
+
+  useEffect(() => {
+    balance &&
+      setHasZeroERC20TokenBalances(
+        Object.keys(balance)?.filter((token) => Number(balance[token]) <= 0)
+          .length > 0
+      )
+    setHasZeroNetworkTokenBalance(Number(nativeBalance?.balance) <= 0)
+  }, [balance, nativeBalance])
+
+  const wrapperClasses = cx({
+    automation: true,
+    enabled: isAutomationEnabled
+  })
+
+  const indicatorClasses = cx({
+    indicator: true,
+    enabled: isAutomationEnabled,
+    warning: hasZeroERC20TokenBalances,
```

Comment:
thought (non-blocking): green and red are pretty clear indicators, but knowing that yellow indicates a zero balance of ocean and EUROe tokens might not be as clear. We could think about adding a warning or some sort of message to give the user a clue of what's happening.

Reply:
I created an issue for this 👍